### PR TITLE
drop subscription laster

### PR DESCRIPTION
**Pull request by Shubham8287**
# Description of Changes

Please describe your change, mention any related tickets, and so on here.

# API and ABI breaking changes

If this is an API or ABI breaking change, please apply the
corresponding GitHub label.

# Expected complexity level and risk

*How complicated do you think these changes are? Grade on a scale from 1 to 5,
where 1 is a trivial change, and 5 is a deep-reaching and complex change.*

*This complexity rating applies not only to the complexity apparent in the diff,
but also to its interactions with existing and future code.*

*If you answered more than a 2, explain what is complex about the PR,
and what other components it interacts with in potentially concerning ways.*

# Testing

*Describe any testing you've done, and any testing you'd like your reviewers to do,
so that you're confident that all the changes work as expected!*

- [x] *Write a test you've completed here.*
- [ ] *Write a test you want a reviewer to do here, so they can check it off when they're satisfied.*

### CODE REVIEW

File: crates/core/src/subscription/module_subscription_actor.rs [hunk 1]
```diff
@@ -86,7 +86,6 @@ impl ModuleSubscriptions {
         subscriptions.remove_subscription(&sender.id.identity);
         subscriptions.add_subscription(sender.clone(), execution_set.into_iter());
         let num_queries = subscriptions.num_queries();
-        drop(subscriptions);
```
Comment (mamcx) [inline crates/core/src/subscription/module_subscription_actor.rs:89]: Here or elsewhere an explanation of why this change?

**Comment by joshua-spacetime** [comment]
Closing as this was transferred to #997.
